ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -19,7 +19,13 @@
     <category field="social_science"/>
     <category field="philosophy"/>
     <category field="literature"/>
-    <updated>2016-11-17T16:34:49+00:00</updated>
+    <category field="anthropology"/>
+    <category field="geography"/>
+    <category field="history"/>
+    <category field="humanities"/>
+    <category field="political_science"/>
+    <category field="sociology"/>    
```

Comment:
this seems a bit excessive in terms of disciplines. Wouldn't social_science and humanities just cover this? Too many categories make this less useful as metadata